app/optimization_model.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+"""model.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1_jxnL3HQvWheA9remErj5oWdwRA-j_e6
+"""
+
+# !pip install gurobipy
+import random
+import gurobipy as grb
+import pandas as pd
+from sqlalchemy import column
+
+
+# Gurobi WLS license file
+# Your credentials are private and should not be shared or copied to public repositories.
+# Visit https://license.gurobi.com/manager/doc/overview for more information.
+WLSACCESSID='6467d903-5c72-46b0-89de-4cc058f8df40'
+WLSSECRET='2e64aa23-f34c-4f10-bf2f-25ec48e06700'
+LICENSEID=770264
+
+# Setup Gurobi License for model
+# Create environment with WLS license
+e = grb.Env(empty=True)
+e.setParam('WLSACCESSID', WLSACCESSID)
+e.setParam('WLSSECRET', WLSSECRET)
+e.setParam('LICENSEID', LICENSEID)
+# e.start()
+
+# Read backend data from github
+intensity_scores = pd.read_csv("https://raw.githubusercontent.com/gaurav613/fydp_bess/main/Data/Intensity_score.csv")
+
+"""Variables: Decision Variables and Parameters"""
+def optimize(CN = 0.17, CM = 0.113, CF = 0.082, UN_m = 500, UM_m = 300, UF_m = 380):
```

Comment:
need to parse the incoming dictionary and assign correct variables for the model.

Reply:
**time of use** dictionary will look as follows:
<img width="143" alt="image" src="https://user-images.githubusercontent.com/52054023/157819706-e7fe14dc-e236-4b63-8c4a-148951af61ee.png">

**Tiered** dictionary will look as follows:
<img width="139" alt="image" src="https://user-images.githubusercontent.com/52054023/157819780-e808b907-90df-4f82-a181-b412ce1d9bea.png">